packages/perseus/src/styles/constants.ts

```diff
@@ -1,7 +1,7 @@
-// @baseFontFamily:        "Proxima Nova", sans-serif;
-export const baseFontFamily = "'Proxima Nova',sans-serif";
-// @boldFontFamily:        "Proxima Nova Semibold", sans-serif;
-export const boldFontFamily = "'Proxima Nova Semibold',sans-serif";
+// @baseFontFamily:        "Lato", sans-serif;
+export const baseFontFamily = "'Lato', sans-serif";
+// @boldFontFamily:        "Lato", sans-serif;
+export const boldFontFamily = "'Lato', sans-serif";
```

Comment:
Should this be some sort of bold-ish variant?  Otherwise the `base` and `bold` font families become identical.

Reply:
I found references to `Lato-Bold` in webapp, so I made that the default for bold here. Falling back to "Lato" just in case though.